crates/cxx-qt-gen/src/parser/property.rs

```diff
@@ -110,20 +114,42 @@ impl ParsedQProperty {
                 let mut write = None;
                 let mut notify = None;
 
+                let mut constant = false;
+                let mut required = false;
+                let mut is_final = false;
+                let mut reset = None;
+
+                let map_auto_or_custom = |variable: &mut Option<FlagState>, value: &Option<Ident>| {
+                    *variable = Some(value.as_ref().map_or(FlagState::Auto, |ident| FlagState::Custom(ident.clone())));
+                };
```

Comment:
if this closure was inside the other closure, then it could probably capture the `value`, but also fine to be outside if you think it's cleaner to have no capturing

Reply:
it did capture it, but it moves it, so then can't be used later in that closure in the reset block

Reply:
Ah makes sense